src/WorkflowManager/ConditionsResolver/Parser/ConditionalParameterParser.cs

```diff
@@ -344,6 +350,24 @@ private void ClearWorkflowParser()
             return null;
         }
 
+        private static string? GetValueFromDictionary(Dictionary<string, string> dictionary, string? key)
+        {
+            if (key is null)
+            {
+                return null;
+            }
+
+            if (dictionary.TryGetValue(key, out var value))
+            {
+                if (value is string valueStr)
```

Comment:
Won't this be redundant since the dictionary is of type `Dictionary<string, string>`?

Reply:
Good point